liferay-tango/liferay-portal
# LPS-171060 A draft version of a page is publicated when saving an AB Test variant

Motivation
=======
There is a bug that causes that a draft version of a page is publicated when saving an AB Test variant

Link to the story: [LPS-171060](https://issues.liferay.com/browse/LPS-171060)

Proposed Solution
========
The draft version to be the same as before the AB Test. 

For that we are going to : 

> Create a new action command to save the variant.
> Add a new success message when that variant is saved:   "The variant was saved successfully."


Steps to Verify:
----------------
1. Create a draft version of a page
2. Create an AB Test for that page
3. Create a variant > Edit the variant (notice that you'll see the draft version already)
4. Click 'Save Variant'

Screenshots:
----------------------


**Note:**
----------------------
The test are  in progress
